src/js/components/HandleSearchError.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
```

Comment:
Created this component for the error handling approach suggested here: https://reactjs.org/blog/2017/07/26/error-handling-in-react-16.html.